include/RAJA/policy/hip/policy.hpp

```diff
@@ -251,9 +263,30 @@ struct hip_atomic_explicit{};
  */
 using hip_atomic = hip_atomic_explicit<seq_atomic>;
 
-using hip_reduce = hip_reduce_base<false>;
+template < bool maybe_atomic,
+           size_t replication = named_usage::unspecified,
+           size_t atomic_stride = named_usage::unspecified,
+           bool init_on_host = false,
```

Comment:
init_on_host and avoid_fences appear in different order here than they do above and below. Please make ordering the same everywhere to avoid confusion (and potential errors).

Reply:
I've already made that error and confused myself so I'm planning to change this to an enum or something.